lib/components/__tests__/clock.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+import { Clock } from '../clock/clock';
+
+describe('clock', () => {
+    it('should render', () => {
```

Comment:
In that case, I would take this file out. I would take anything that isn't driving your point that you're trying to address with this repo.